app/models/s3-uploader-db-schema.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var mongoose = require('mongoose');
+
+var s3UploaderDbSchema = mongoose.Schema({
+  Category: {
+    type: String,
+    required: true
+  },
+  CategoryID: {
+    type: String,
+    required: true
+  },
+  MetaTitle: String,
+  MetaDescription: String
+});
```

Comment:
Please rename the keys with respect to naming conventions. For eg:
**Category** should be renamed as **category**